src/Microsoft.Dnx.Compilation.CSharp.Common/project.json

```diff
@@ -16,6 +23,7 @@
         },
         "dnxcore50": {
             "dependencies": {
+                "System.Console": "4.0.0-beta-*",
```

Comment:
Yes, it for Logger which is part of `Dnx.Runtime.Sources`(which this project references)